src/routes/auth.py

```diff
@@ -83,8 +83,6 @@ async def refresh_token(credentials: HTTPAuthorizationCredentials = Security(oau
         token = credentials.credentials
     else:
         token = credentials
-async def refresh_token(credentials: HTTPAuthorizationCredentials = Security(oauth2_scheme), db: Session = Depends(get_db)):
```

Comment:
The code in this refresh_token function is old and doesn't work properly. Please make sure you have the proper refresh_token function. Also, why did you remove these 2 lines?

`async def refresh_token(credentials: HTTPAuthorizationCredentials = Security(oauth2_scheme),
                        db: Session = Depends(get_db)):
    if hasattr(credentials, 'credentials'):
        token = credentials.credentials
    else:
        token = credentials
    email = await auth_service.decode_refresh_token(token)
    user = await repository_users.get_user_by_email(email, db)
    if user.refresh_token != token:
        await repository_users.update_token(user, None, db)
        raise HTTPException(status_code=status.HTTP_401_UNAUTHORIZED, detail="Invalid refresh token")

    access_token = await auth_service.create_access_token(data={"sub": email})
    refresh_token = await auth_service.create_refresh_token(data={"sub": email})
    await repository_users.update_token(user, refresh_token, db)
    return {"access_token": access_token, "refresh_token": refresh_token, "token_type": "bearer"}`